src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/Listeners/StorageBlobContainerComparer.cs

```diff
@@ -3,14 +3,14 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.Azure.WebJobs.Host.Storage.Blob;
```

Comment:
Rename file to CloudBlobContainerComparer

Reply:
done